### DISCUSSION TITLE
DISCUSSION: January 2025  DataFusion Meetup in Amsterdam / CIDR 2025

### DISCUSSION THREAD

**Discussion by alamb** [apache/datafusion#12988] [General]
I would like to dicuss having a DataFusion meetup in Amsterdam , either before/after https://www.cidrdb.org/cidr2025/
 (January 19-22, 2025)

There will likely be many interested parties already in Amsterdam for this conference (including myself, as I am a coauthor on a paper accepted at CIDR this year)

I can help organize the content / speakers, but I need help finding a suitable space / Venue. 

Previously @kavirajk  and @Dtenwolde mentioned that they had some ideas https://github.com/apache/datafusion/discussions/10347#discussioncomment-10139861

Is anyone else interested in helping or might have ideas on how to organize a space / venue for this meetup?

**Comment by alamb** [comment]
I believe @SamSynnada may be interested in this as well

**Comment by SamSynnada** [comment]
Spoke with a friend from Xomnia, @backkem is also looking for a venue

**Reply by alamb** [reply]
Maybe @Dandandan  who I think is also based in the Netherlands can suggest other areas to explore

**Reply by alamb** [reply]
I wonder if @Abdullahsab3  / their team are perhaps based near this venue given their previous comment https://github.com/apache/datafusion/issues/11442#issuecomment-2361442745

**Reply by Abdullahsab3** [reply]
We're based in Belgium (Ghent), couple of hours away from Amsterdam. I still have the meetup on my mind but had a very busy couple of weeks. I spoke about it with my team. My CTO mentioned a while ago that he may know some places that are suitable for a meetup in Amsterdam, but he wonders whether there is a specific budget. I thought perhaps there might be already datafusion users in Amsterdam who might have a place to host the meetup instead :p

**Reply by alamb** [reply]
Thanks @Abdullahsab3  -- indeed I think we should try and find someone who has space that can host the event rather than paying for it. Sounds like maybe @SamSynnada   has worked some magic. Let's see

**Comment by ozankabak** [comment]
If I'm not mistaken, we made some progress on this. @SamSynnada will probably write an update soon

**Comment by SamSynnada** [comment]
Had a talk with Xomnia, my notes + questions:

- Xomnia is a data/AI consulting company from NL that incubated Polars. Working with Sarah from marketing there.
- They want to make this part of their [Data & Drinks](https://www.xomnia.com/event-series/data-and-drinks/) series which I think is great - they pull in a solid technical crowd at these events.
- [Venue](https://maps.app.goo.gl/Gh2M6iCe2VHW9BGG8) looks good too, it's in Amsterdam (central-ish).
- We've got the space from 17:30-21:00, with Xomnia handling food/drinks. They usually do about an hour of talks.
- I told them we should have at least 4 speakers (we could make this 5+?). I was thinking @alamb, @Dandandan, @ozankabak. 

Main things to discuss
- Any comments on date, timing, Xomnia? (@alamb: could we do Jan 22? > Will check with Xomnia)
- Let's clarify the speaker line-up. Really need community guidance on this, I don't know who will be there for CIDR.

**Reply by alamb** [reply]
THANK YOU @SamSynnada !

So I am hoping we can do this an end of CIDR 2025 https://www.cidrdb.org/cidr2025/ to take advantage of everyone who will be in town already for the conference (like myself).

Thus I think Wed Jan 22 would be a good choice as the conference ends earlier that day (if it follows the agenda of cidr 2023)

**Reply by SamSynnada** [reply]
just spoke with xomnia—they’re available on the Jan 15th or 23rd. which one works better?

**Reply by alamb** [reply]
Jan 23 I think is more likely we can piggyback on the existing CIDR conference attendees

**Reply by SamSynnada** [reply]
Jan 23rd confirmed 👍 let's focus on the line-up.

**Reply by Dandandan** [reply]
I can confirm to prepare / give a talk that day. Thanks for organizing / setting this up @SamSynnada

**Reply by kavirajk** [reply]
Thanks @SamSynnada. Looking forward to it. Are there any way to RSVP to the meetup?

**Reply by alamb** [reply]
> Thanks @SamSynnada. Looking forward to it. Are there any way to RSVP to the meetup?

Since the event will be run by Xomnia, I believe we are waiting on them to create the event / signup page

**Comment by SamSynnada** [comment]
We’ve got @alamb, @ozankabak, and @Dandandan confirmed as speakers. Who else can we add to the list? Also, we’ll need the titles of the talks so we can start promoting the event with the Xomnia team.

**Comment by JanKaul** [comment]
If we don't have enough speakers, I could give a talk about using Iceberg Materialized Views with Datafusion.

**Reply by alamb** [reply]
YES! that would be awesome -- i will add you to the list of speakers

**Reply by alamb** [reply]
@JanKaul  -- Can you send me an email at `alamb@influxdata.com` and I'll get you setup on the list?

**Comment by SamSynnada** [comment]
Our host, Xomnia, has a strict format of two or three speakers in their own events. Initially, they made an exception for us and allowed four, so we planned for four speakers, but now my contact tells me three is a hard limit and she has problems selling four speakers internally. We’ll need to drop someone. Anyone volunteering to drop their talk? If not we will decide on a first-come, first-served basis. 

@JanKaul @Dandandan @ozankabak @alamb

**Reply by alamb** [reply]
I am fine with anything. Thank you for driving this @SamSynnada

**Reply by Dandandan** [reply]
Feel free to drop my talk, I didn't prepare anything yet for the topic and am pretty busy privately.

**Reply by SamSynnada** [reply]
@JanKaul could you review your profile, talk description:

https://docs.google.com/document/d/1AH9cYGw-BGdcbVzmqBMktm2WecLEDyenCztv__6w3AU/edit?tab=t.0

**Reply by JanKaul** [reply]
It's too bad there is not enough time for all talks. Thanks @Dandandan for stepping down, I'm really excited to give a talk and otherwise it would have been my talk. Thanks @SamSynnada for the organization, I've updated my profile.

**Comment by alamb** [comment]
The official signup link is now available: 

https://www.meetup.com/data-drinks/events/304895184/

**Comment by JanKaul** [comment]
I didn't find the length of the talks anywhere. In the schedule there is one hour for all talks. Does that mean each speaker has 20 mins including questions?

**Reply by mbrobbel** [reply]
The linked meetup page has an agenda:

> Agenda:
17:30-18.30: Walk-ins & dinner
18.30-18.35: Introduction to Xomnia
18:35-18:55: Intro to DataFusion: Technology, Community, and Not Quite Enough Time by Andrew Lamb
18:55-19:00: Break
19:00-19:25: Building A Unified Compute Engine with Apache DataFusion by Mehmet Ozan Kabak
19:25-19:35: Break
19:35-20.00: Distributed Joins with DataFusion at Coralogix
20:00-21.00: Borrel

**Reply by JanKaul** [reply]
Ah thanks for the schedule. The only thing is that the last talk "Distributed Joins with Datafusion at Coralogix" has been swapped to my Talk: "Iceberg Materialized Views with Datafusion".

**Reply by alamb** [reply]
Yes, that is correct -- There was time only for 3 talks. 🚀

**Comment by alamb** [comment]
Here are some pictures of the event (thanks @jakthom) 

I believe the talks will be posted online shortly.

My slides are here: [2025-01-23 Intro to DataFusion: Technology, Community, and Not Quite Enough Time](https://docs.google.com/presentation/d/1Qm15RbzG77GKah6_FDRFFPSpvxWw0zDoGrH5Dz2e_2w/edit#slide=id.p)

![PXL_20250123_173125235](https://github.com/user-attachments/assets/b0c0af3e-4aa8-470b-936d-dd7ae9846783)
![PXL_20250123_173425345](https://github.com/user-attachments/assets/1524db4d-0e5b-4fec-b726-ba070f6edd39)
![PXL_20250123_180905212 MP](https://github.com/user-attachments/assets/a454d975-8202-44f8-80b5-996fa43d0cb1)
![PXL_20250123_183340319 MP](https://github.com/user-attachments/assets/6f915767-dc5a-42ea-83f8-d0da6fbea23e)
![PXL_20250123_183720247 MP](https://github.com/user-attachments/assets/042f0688-ba78-421c-8498-aee27a8b4a33)
![PXL_20250123_184257784](https://github.com/user-attachments/assets/0269a7e7-02be-46bd-9a05-22bd81c07408)

**Comment by alamb** [comment]
Recording of the talks are here: https://www.youtube.com/watch?v=GruBeVDoWq4

**Comment by JanKaul** [comment]
The slides of my talk are here: https://www.dropbox.com/scl/fi/pkc9b7yo46dul0j220t1p/ivm.pdf?rlkey=no3ju1zo1eo8zchb2og8kgyce&e=1&st=oyxppacn&raw=1

**Reply by alamb** [reply]
Mine are here: https://docs.google.com/presentation/d/1Qm15RbzG77GKah6_FDRFFPSpvxWw0zDoGrH5Dz2e_2w/edit#slide=id.p

[2025-01-23 Intro to DataFusion_ Technology, Community, and Not Quite Enough Time.pdf](https://github.com/user-attachments/files/18628665/2025-01-23.Intro.to.DataFusion_.Technology.Community.and.Not.Quite.Enough.Time.pdf)

**Comment by oznur-synnada** [comment]
# Recap of the Event

We’ve wrapped up the first [Data & Drinks](https://www.meetup.com/data-drinks/events/304895184/) event of 2025 which featured talks on Apache DataFusion, including real-world applications and a deep dive into the technology’s inner workings. Three DataFusion contributors (@alamb, @ozankabak, @JanKaul) took the stage.

The event, hosted by [Xomnia](https://www.xomnia.com/) in their HQ, took place in Amsterdam on January 23, 2025. Being the exclusive focus of an event series which pulls in a technical crowd was a huge moment for the DataFusion community in Europe. The interest of the crowd led to fruitful discussions about current and future use cases of DataFusion. All in all, we left Amsterdam with great new connections and inspiration!

Full recording of the event: [Data & Drinks: Building Next-Gen Data Systems with Apache DataFusion](https://www.youtube.com/watch?v=GruBeVDoWq4&ab_channel=Xomnia)

## Talks

Apache DataFusion PMC Chair @alamb from InfluxDB delivered the first talk of the event and delved into the architecture, modularity, and tradeoffs of Apache DataFusion, and how it's employed in building advanced data systems.
![406729168-1524db4d-0e5b-4fec-b726-ba070f6edd39](https://github.com/user-attachments/assets/fcaf01c7-858e-44b2-aee0-8c51ddece69d)


Next, it was Synnada’s Co-Founder & CEO, Apache DataFusion PMC @ozankabak's turn to talk about how DataFusion’s modular architecture enables the vision of unified compute engines. 
![406729169-a454d975-8202-44f8-80b5-996fa43d0cb1](https://github.com/user-attachments/assets/d9f47f06-d3f7-4ca4-8357-f2c84e98b43b)


Lastly, @JanKaul, the Founder & CEO of Dashbook, explored the implementation of materialized views in Apache Iceberg using DataFusion. 
![406729174-0269a7e7-02be-46bd-9a05-22bd81c07408](https://github.com/user-attachments/assets/c13dd7e9-44a5-4b90-a8d5-9b3ed3cb19b3)


## Closing Remarks

- We have seen that the real-world applications and cutting-edge innovations of Apache DataFusion receive growing attention from the data ecosystem in Europe.
- 110+ attendees RSVP’d for the event.
- Many thanks to [Xomnia](https://www.xomnia.com/) for hosting, them handling the venue and servings made it much easier for us to focus on the talks and discussions.

**Comment by ohommos** [comment]
Hi all, thanks for organising the meetup back in January.
For next time, my company Adyen would be happy to host. We have a venue that’s enough for ~50 and another for ~120. When there’s enough interest for round 2, please let me or @santosh-d3vpl3x know and we can help arrange!

**Reply by oznur-synnada** [reply]
> Hi all, thanks for organising the meetup back in January. For next time, my company Adyen would be happy to host. We have a venue that’s enough for ~50 and another for ~120. When there’s enough interest for round 2, please let me or @santosh-d3vpl3x know and we can help arrange!

Thank you for your help @ohommos - here is the discussion for the May 2025 Amsterdam meetup:  https://github.com/apache/datafusion/discussions/16038
